js/views/RegistrationPages.js

```diff
@@ -123,7 +123,8 @@ const RegistrationPages = React.createClass ({
    */
   beforePageChange(currentPage, nextPage) {
     const shouldContinue = this.validatePage();
-    if (shouldContinue || nextPage < currentPage) {
+    if (shouldContinue || nextPage >= 0 && nextPage < currentPage) {
```

Comment:
That `this.setIndex` method in this view is being called every time the swiper is allowed to switch to a new page, if the swiper attempts to move backwards to index -1 it can lock the view.

This is a tiny fix that prevents that from happening.